docs/usecases/deployment_without_ocs.md

```diff
@@ -27,7 +27,28 @@ names separated by a space in `CLUSTER_CONF` field. Eg.:
 conf/deployment/aws/ipi_1az_rhcos_lso_3m_3w.yaml conf/ocsci/skip_ocs_deploy.yaml
 ```
 
-## Enabling catalog source with development builds of OCS
+## Catalog sources with CI builds
+
+Note that this section assumes you have your environment configured as if you
+were running ocs-ci tests (`oc` can connect to your cluster, pull secret
+provided to `openshift-installer` by ocs-ci during OCP deployment included one
+for `quay.io/rhceph-dev` and `brew.registry.redhat.io`).
```

Comment:
@mbukatov brew.registry.redhat.io is needed only when you deploy LSO with non GA version of OCP. Currently OCP 4.7, but once it's GAed it's not needed and will be needed for OCP 4.8. So maybe worth to mention it here.

Reply:
Also worth to mention that if using custom catalog source - we rely on this label for ocs internal build:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/templates/ocs-deployment/catalog-source.yaml#L7-L8

To avoid conflict with packageManifest from GAed version from redhat-operators source.

Reply:
@clacroix12 not sure if you are using some other label for LSO? I remember you did the similar approach there, right?

Reply:
@petr-balogh @mbukatov the selector we rely on when using the custom catalog source for non-GA LSO is [defined in our constants module](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L594).

`OPTIONAL_OPERATORS_SELECTOR = "catalog=optional-operators"`

Reply:
Hmm, I could move the info about particular secrets down into the sections bellow.

Reply:
Is the note about labeling catalog source actionable in context of this usecase? What do I need to mention exactly? Also: is it mentioned elsewhere already?

Reply:
I am afraid that we don't have this mentioned anywhere @mbukatov .

The note should be done for manual deployment of OCS internal build somewhere about line 71 in this PR where is mentioned:
```
$ oc create -f ocs_ci/templates/ocs-deployment/local-storage-optional-operators.yaml
```

We can emphasize that label 
```
ocs-operator-internal: "true"
```
Is required as our automation rely on this to properly identify packagemanifest from which we identify current CSV name for installed version.

For LSO we rely on the the optional-operators selector so what we have defined in mentioned file:
local-storage-optional-operators.yaml.

So as you are mentioning files directly we maybe do not have to confuse people about this but they have to be sure they used the same template we provided and not custom catalog sources with different name or labell